NitroxPatcher/Helper/SupportHelper.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace NitroxPatcher.Helper;
+
+internal static class SupportHelper
+{
+    /// <summary>
+    ///     Dependency names commonly used by Unity game mods.
+    /// </summary>
+    private static readonly string[] modDependencies = ["HarmonyX", "0Harmony", "Harmony", "MonoMod.RuntimeDetour"];
+
+    /// <summary>
+    ///     Logs a list of all other mods that are currently loaded.
+    /// </summary>
+    public static string? GetSummaryOfOtherMods()
+    {
+        Assembly[] otherMods = GetOtherMods().ToArray();
+        if (otherMods.Length < 1)
+        {
+            return null;
+        }
+        StringBuilder sb = new();
+        sb.AppendLine("Nitrox has detect other mods! Please do not report Nitrox issues while mods are loaded!");
```

Comment:
```suggestion
        sb.AppendLine("Nitrox has detected other mods! Please do not report Nitrox issues while mods are loaded!");
```
Minor spelling suggestion :)